api/src/services/settings.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
 
         if (JSON.stringify(doc.settings) !== original) {
           info('Updating settings with new defaults');
-          return db.medic.put(doc);
+          return db.medic.put(doc).then(() => true);
         }
 
         return Promise.resolve();
```

Comment:
Let's resolve `false` here to be explicit. It's probably worth a log output too otherwise you can't tell that nothing happened...

```suggestion
        info('Not updating settings - the existing settings are already up to date');
        return false;
```

Reply:
Done!